net/src/ipc/spawn.rs

```diff
@@ -158,6 +161,22 @@ pub fn ipc_spawn(
     Ok(out)
 }
 
+#[cfg(windows)]
+fn term_child(_pid: u32) -> bool {
+    false
+}
+
+#[cfg(not(windows))]
+fn term_child(pid: u32) -> bool {
+    unsafe {
+        if libc::kill(pid as i32, libc::SIGTERM) == 0 {
+            libc::waitpid(pid as i32, std::ptr::null_mut(), 0);
+            return true;
+        }
```

Comment:
What about when kill returns nonzero? Are we concerned about this cases at all, at least perhaps from a logging perspective?

```
The kill() function shall fail if:

[EINVAL]
The value of the sig argument is an invalid or unsupported signal number.
[EPERM]
The process does not have permission to send the signal to any receiving process.
[ESRCH]
No process or process group can be found corresponding to that specified by pid.
```

There is similar error handling possible with `waitpid`, but again not sure how important it is to check for it here.

Reply:
Yeah, it's not very robust, but if the SIGTERM fails, it should still send the KILL.

Added an issue to implement a more robust solution in the future: https://github.com/holochain/holochain-rust/issues/1357